src/modules/dashboard/service/dashboardService.ts

```diff
@@ -845,15 +855,18 @@ export class DashboardService {
                 
                 transactionDetails.displayTips.push(newRowTip);
             }
+
         }
 
         let data = {
             transferList: transfer,
             recipient: sendTo,
             recipientType: toType,
+            recipientName: this.addressConvertToName(sendTo),
             messageTypeString: messageTypeString,
             messageType: messageType,
             messagePayload: transferTx.message.payload,
+            amount: txnAmount?Helper.convertToExact(txnAmount, 6):'',
```

Comment:
later will need to change, for the divisibility

Reply:
I see, noted. Shall we leave it first till you rework the dashboard script?

Reply:
ya, can leave it first